Description: rename libvirt group name to 'libvirtd'
Author: Soren Hansen <soren@ubuntu.com>
Forwarded: no

Index: ubuntu-libvirt/daemon/libvirtd.conf
===================================================================
--- ubuntu-libvirt.orig/daemon/libvirtd.conf
+++ ubuntu-libvirt/daemon/libvirtd.conf
@@ -87,7 +87,7 @@
 # without becoming root.
 #
 # This is restricted to 'root' by default.
-unix_sock_group = "libvirt"
+unix_sock_group = "libvirtd"
 
 # Set the UNIX socket permissions for the R/O socket. This is used
 # for monitoring VM status only
