Index: libvirt/daemon/libvirtd.service.in
===================================================================
--- libvirt.orig/daemon/libvirtd.service.in
+++ libvirt/daemon/libvirtd.service.in
@@ -17,8 +17,9 @@ ExecStart=@sbindir@/libvirtd $libvirtd_o
 ExecReload=/bin/kill -HUP $MAINPID
 KillMode=process
 Restart=on-failure
-# Override the maximum number of opened files
-#LimitNOFILE=2048
+LimitNOFILE=infinity
+LimitNPROC=infinity
+TasksMax=infinity
 
 [Install]
 WantedBy=multi-user.target
